### PR TITLE
filter out broken benchmark

### PR DESCRIPTION
**Pull request by jhoenicke**
Use the broken_fp list to remove benchmarks from the all and new lists.

### CODE REVIEW

File: 2020/prep/filter_benchmarks.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+filter_benchmark () {
```
Comment (HanielB) [inline 2020/prep/filter_benchmarks.sh:3]: Could you add documentation to what the script is doing?

Comment (jhoenicke) [reply 2020/prep/filter_benchmarks.sh:10]: done.

**Comment by jhoenicke** [comment]
Antti's change looks fine to me.  Should I merge this pull request?

**Comment by aehyvari** [comment]
Go ahead please!

**Pull request merged**